tests/integration/test_cladetime_integration.py

```diff
@@ -17,46 +17,47 @@
 
-    with freeze_time("2024-11-01"):
-        ct = CladeTime()
+    ct = CladeTime()
 
-        metadata_filtered = sequence.filter_metadata(ct.sequence_metadata, collection_min_date="2024-10-01")
+    metadata_filtered = sequence.filter_metadata(ct.sequence_metadata, collection_min_date="2024-10-01")
 
-        # store clade assignments as they exist on the metadata file downloaded from Nextstrain
-        original_clade_assignments = metadata_filtered.select(["strain", "clade"])
+    # store clade assignments as they exist on the metadata file downloaded from Nextstrain
+    original_clade_assignments = metadata_filtered.select(["strain", "clade"])
 
-        # assign clades to the same sequences using cladetime
-        assigned_clades = ct.assign_clades(metadata_filtered, output_file=assignment_file)
+    # assign clades to the same sequences using cladetime
+    assigned_clades = ct.assign_clades(metadata_filtered, output_file=assignment_file)
+
+    # clade assignments via cladetime should match the original clade assignments
+    check_clade_assignments = original_clade_assignments.join(
+        assigned_clades.detail, on=["strain", "clade"]
+    ).collect()
+    assert len(check_clade_assignments) == len(metadata_filtered.collect())
+    unmatched_clade_count = check_clade_assignments.filter(pl.col("clade").is_null()).shape[0]
+    assert unmatched_clade_count == 0
+
+    # summarized clade assignments should also match summarized clade assignments from the
+    # original metadata file
+    assert_frame_equal(
+        sequence.summarize_clades(metadata_filtered.rename({"clade": "clade_nextstrain"})),
+        assigned_clades.summary,
+        check_column_order=False,
+        check_row_order=False,
+    )
 
-        # clade assignments via cladetime should match the original clade assignments
-        check_clade_assignments = original_clade_assignments.join(
-            assigned_clades.detail, on=["strain", "clade"]
-        ).collect()
-        assert len(check_clade_assignments) == len(metadata_filtered.collect())
-        unmatched_clade_count = check_clade_assignments.filter(pl.col("clade").is_null()).shape[0]
-        assert unmatched_clade_count == 0
-
-        # summarized clade assignments should also match summarized clade assignments from the
-        # original metadata file
-        assert_frame_equal(
-            sequence.summarize_clades(metadata_filtered.rename({"clade": "clade_nextstrain"})),
-            assigned_clades.summary,
-            check_column_order=False,
-            check_row_order=False,
-        )
-
-        # metadata should reflect ncov metadata as of 2024-11-01
-        assert assigned_clades.meta.get("sequence_as_of") == datetime(2024, 11, 1, tzinfo=timezone.utc)
-        assert assigned_clades.meta.get("tree_as_of") == datetime(2024, 11, 1, tzinfo=timezone.utc)
-        assert assigned_clades.meta.get("nextclade_dataset_version") == "2024-10-17--16-48-48Z"
```

Comment:
Noting that I think we could hard-code this info as seen in the 2024-10-30 version of the hub metadata?
https://github.com/reichlab/variant-nowcast-hub/blob/main/auxiliary-data/modeled-clades/2024-10-30.json